### ISSUE TITLE
When editing app show if synchronized

### ISSUE THREAD

**Issue by odinuv** [keboola/developer-portal-ui#360]
use case - when deploying a new app version, there is no way to tell if it is already synchronized to all KBCs or not (other then runnning the component which is greatly annoying).

solution - in app detail ping Storage API and check whether it is current?
might need a sync timestamp in Storage API response to reliably detect whether it is current?

**Comment by Halama** [comment]
It's a good idea. I would add sync of developer portal app version to KBC components list for accurate detection https://connection.keboola.com/v2/storage .

**Comment by ujovlado** [comment]
Duplicate of #327

**Comment by kacurez** [comment]
imho 2 timestamps are required - one from the sapi as last sync time and the other representing last change of component(maybe latest version timestamp). I think none of the timestamps is currently available.

**Comment by ujovlado** [comment]
Looks like change mentioned in https://github.com/keboola/developer-portal-ui/issues/360#issuecomment-343506652 should be enough.

After app save (in dev portal) we can compare version number returned (saved in state) with version from connection.

**Comment by Halama** [comment]
Yes not timestamps are required, just dev portal app version in KBC.
The dev portal UI will poll the KBC stacks and will just compare the versions and show if app is in sync.

**Comment by Halama** [comment]
SAPI updatet deployed https://connection.keboola.com/v2/storage :
![image](https://user-images.githubusercontent.com/903531/33024315-0c5eae06-ce0b-11e7-9115-24d8caea5915.png)

**Comment by odinuv** [comment]
uaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by ujovlado** [comment]
pod stromcek? :christmas_tree:

**Comment by odinuv** [comment]
https://keboolaglobal.slack.com/archives/C5K31TYKT/p1545634596009100

**Comment by odinuv** [comment]
https://keboola.slack.com/archives/C02CGK5CL/p1556281584009400?thread_ts=1556280540.007300&cid=C02CGK5CL
